integration-scripts/src/deployV0.5Contracts.ts

```diff
@@ -36,9 +42,22 @@ export async function deployContracts() {
     name: 'MeanAggregator',
   })
 
+  // deploy PrepaidAggregator
```

Comment:
I think we can :scissors: this, the code line is self explanatory

Reply:
:+1: